.coveragerc

```diff
@@ -36,6 +36,7 @@ omit =
     homeassistant/components/apache_kafka/*
     homeassistant/components/apcupsd/*
     homeassistant/components/apple_tv/*
+    homeassistant/components/apprise/*
```

Comment:
You've provided tests, why exclude it from coverage?

Reply:
Doh! I have no excuse; I think i got caught up of searching for other components and just copy/pasting their examples. I'll fix this tonight; good catch.